### PR TITLE
Update SOP for deployments

### PR DESCRIPTION
**Pull request by jacobwindsor**
Updates relating to [#257](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/257)

### CODE REVIEW

**Review by MightyAx — CHANGES_REQUESTED** [review]
Great!
One requested addition:
It would also be useful to know how to redeploy the current production deployment in GitLab from an operational point of view. Can you include the instructions for doing this either here or replacing the outdated Retry-Failed-Bundle-Exporting.md

**Comment by jacobwindsor** [comment]
@MightyAx Done! Thanks for the suggestion

**Review by MightyAx — APPROVED** [review]
That's brilliant thanks

File: docs/operations_tasks/Ingest-Release-SOP.md
Comment (amnonkhen) [inline docs/operations_tasks/Ingest-Release-SOP.md:34]: Can a link be added here? or a notice about the slack channel, and how to join it?

**Review by amnonkhen — CHANGES_REQUESTED** [review]
Very good SOP. Detailed and clear. Only a few minor comments.

File: docs/operations_tasks/Ingest-Release-SOP.md
Comment (amnonkhen) [inline docs/operations_tasks/Ingest-Release-SOP.md:32]: Isn't this done in the github web ui?

File: docs/operations_tasks/Ingest-Release-SOP.md
Comment (amnonkhen) [inline docs/operations_tasks/Ingest-Release-SOP.md:49]: Let's make it clear that this is an emergency procedure. We should still make sure the PR is happening.

File: docs/operations_tasks/Ingest-Release-SOP.md
Comment (amnonkhen) [inline docs/operations_tasks/Ingest-Release-SOP.md:51]: Can this be re-enabled?
If not, it means that following an emergency deployment, we will not return to normal operations, but need to manually deploy from then on?

File: docs/operations_tasks/Retry-Failed-Bundle-Exporting.md
Comment (amnonkhen) [inline docs/operations_tasks/Retry-Failed-Bundle-Exporting.md:7]: What does "bundle" mean in this context?

File: docs/operations_tasks/Retry-Failed-Bundle-Exporting.md
Comment (amnonkhen) [inline docs/operations_tasks/Retry-Failed-Bundle-Exporting.md:15]: This should not be done without approval from a consortium of devs.

File: docs/operations_tasks/Ingest-Release-SOP.md
Comment (jacobwindsor) [reply docs/operations_tasks/Ingest-Release-SOP.md:32]: No. There's a command below

File: docs/operations_tasks/Ingest-Release-SOP.md
Comment (jacobwindsor) [reply docs/operations_tasks/Ingest-Release-SOP.md:51]: Yes, a deployment via gitlab after a manual deploy will return things to a normal state. I will make that clearer

File: docs/operations_tasks/Retry-Failed-Bundle-Exporting.md
Comment (jacobwindsor) [reply docs/operations_tasks/Retry-Failed-Bundle-Exporting.md:7]: Good question - wasn't my writing. I suppose "submissions"

Comment (amnonkhen) [reply docs/operations_tasks/Retry-Failed-Bundle-Exporting.md:7]: A consistent set of terms would help readers and maintainers of this doc.

Comment (jacobwindsor) [reply docs/operations_tasks/Retry-Failed-Bundle-Exporting.md:7]: I think bundles makes sense here actually. Exporting is done on a per experiment basis not a submission basis so it's not really the submission envelope.

Comment (aaclan-ebi) [reply docs/operations_tasks/Retry-Failed-Bundle-Exporting.md:7]: no one speaks of the b* word now, let's just use "experiment" :)))

Comment (jacobwindsor) [reply docs/operations_tasks/Retry-Failed-Bundle-Exporting.md]: Done!

File: docs/operations_tasks/Ingest-Release-SOP.md
Comment (amnonkhen) [inline docs/operations_tasks/Ingest-Release-SOP.md:83]: Here we have a few shell compatibility issues, don't we? If so, please point them out here.

Comment (jacobwindsor) [reply docs/operations_tasks/Ingest-Release-SOP.md:83]: Those issues are well documented at https://github.com/ebi-ait/ingest-kube-deployment/tree/pop+zsh. I don't think duplicating this will help as it increases the chance of things going out of date

Comment (jacobwindsor) [reply docs/operations_tasks/Ingest-Release-SOP.md:83]: Besides, the shell compatibility issues are not relevant to GitLab

File: docs/operations_tasks/Retry-Failed-Bundle-Exporting.md
Comment (aaclan-ebi) [inline docs/operations_tasks/Retry-Failed-Bundle-Exporting.md:10]: There's a less disruptive way to identify and delete the failed pods. I'll add the details later.

**Review by aaclan-ebi — APPROVED** [review]
LGTM!

File: docs/operations_tasks/Retry-Failed-Bundle-Exporting.md
Comment (jacobwindsor) [reply docs/operations_tasks/Retry-Failed-Bundle-Exporting.md:10]: @aaclan-ebi in another PR or this one?

**Review by amnonkhen — APPROVED** [review]

File: docs/operations_tasks/Retry-Failed-Bundle-Exporting.md
Comment (aaclan-ebi) [reply docs/operations_tasks/Retry-Failed-Bundle-Exporting.md:10]: I can add it and create another PR later.

**Pull request merged**
